andela/caret-bn-backend
# #168781676-email signup verification feature

#### What does this PR do?

- add user verify email

#### Description of Task to be completed?

- created email verification api

#### How should this be manually tested?

- Clone the repository from [here](https://github.com/andela/caret-bn-backend)
- got to to Ft-signup-email-168781676 branch
- Add SECRET_KEY variable to .env file
- Run npm install
- run sequelize db:migrate
- run sequelize db:seed:all
- run npm run dev

- in POSTMAN got http://localhost:3000/api/v1/users/register
- go to your e-mail and click verify email button

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

[#168781676](https://www.pivotaltracker.com/story/show/168781676)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/38130305/66665364-ae74b880-ec4e-11e9-82cb-38c69e853d3b.png)


<img width="796" alt="Screen Shot 2019-10-10 at 18 20 38" src="https://user-images.githubusercontent.com/38130305/66587443-b3703400-eb8a-11e9-9262-48f906bfd2ff.png">

#### Questions:
- N/A
